hack/e2e.sh

```diff
@@ -77,10 +77,18 @@ export E2E_TESTDATA_DIR
 read -r E2E_METADATA_REVISION <"${E2E_TESTDATA_DIR}"/metadata_revision
 export E2E_METADATA_REVISION
 
+CI_REGISTRY_URL="registry.ci.openshift.org/cincinnati-ci-public/cincinnati-graph-data"
+
+if [[ ! -z "${GRAPHDATA_IMAGE}" ]]; then
```

Comment:
```suggestion
export GRAPH_SOURCE="${GRAPHDATA_IMAGE:-registry.ci.openshift.org/cincinnati-ci-public/cincinnati-graph-data:${E2E_METADATA_REVISION}}"
```
Similar to what we do in [cincinnati-operator](https://github.com/openshift/cincinnati-operator/blob/master/hack/deploy.sh#L10)